samples/OpenIdConnectSample/Readme.md

```diff
@@ -0,0 +1,44 @@
+# How to set up the sample locally
+
+The OpenIdConnect sample support multilpe authentication providers. In this instruction, we will explore how to set up this sample with both Azure Active Directory and Google Firebase Authentication
```

Comment:
I don't think it's specific to Firebase, I think it's a general Google login thing. Or there was some weird rebranding.